extensions/iidm/src/LoadDetailXmlSerializer.cpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <powsybl/iidm/extensions/iidm/LoadDetailXmlSerializer.hpp>
+
+#include <cmath>
+
+#include <powsybl/iidm/Load.hpp>
+#include <powsybl/iidm/converter/xml/NetworkXmlReaderContext.hpp>
+#include <powsybl/iidm/converter/xml/NetworkXmlWriterContext.hpp>
+#include <powsybl/iidm/extensions/iidm/LoadDetail.hpp>
+#include <powsybl/iidm/extensions/iidm/LoadDetailAdder.hpp>
+#include <powsybl/stdcxx/make_unique.hpp>
+#include <powsybl/stdcxx/math.hpp>
+#include <powsybl/xml/XmlStreamReader.hpp>
+#include <powsybl/xml/XmlStreamWriter.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace extensions {
+
+namespace iidm {
+
+LoadDetailXmlSerializer::LoadDetailXmlSerializer() :
+    converter::xml::AbstractExtensionXmlSerializer("detail", "network", "ld", "http://www.itesla_project.eu/schema/iidm/ext/load_detail/1_0") {
+}
+
+Extension& LoadDetailXmlSerializer::read(Extendable& extendable, converter::xml::NetworkXmlReaderContext& context) const {
+    if (!stdcxx::isInstanceOf<Load>(extendable)) {
+        throw AssertionError(stdcxx::format("Unexpected extendable type: %1% (%2% expected)", stdcxx::demangle(extendable), stdcxx::demangle<Load>()));
+    }
+    double fixedActivePower = context.getReader().getOptionalAttributeValue("fixedActivePower", stdcxx::nan());
```

Comment:
Notice : before, we read "subLoad1ActivePower" attribute, and if absent, read attribute "fixedActivePower" (same for other attributes) : 
```
    double fixedActivePower = context.getReader().getOptionalAttributeValue("subLoad1ActivePower", stdcxx::nan());
    if (std::isnan(fixedActivePower)) {
        fixedActivePower = context.getReader().getAttributeValue<double>("fixedActivePower");
    }
```